src/LeanplumRequest.ts

```diff
@@ -103,7 +108,7 @@ export default class LeanplumRequest {
       return
     }
 
-    let sendNow = InternalState.devMode || options.sendNow || !LeanplumRequest.batchEnabled
+    let sendNow = options.devMode || options.sendNow || !LeanplumRequest.batchEnabled
```

Comment:
This is the condition that will evaluate to false if `sendNow` is not set in production.

Reply:
Thanks, it's great that you noticed it on time.